.github/workflows/test.yml

```
@@ -91,6 +91,12 @@ jobs:
         # Custom app-config.local.yaml
         cp -vr configs/app-config/app-config.local.example.yaml configs/app-config/app-config.local.yaml
 
+    - name: Create dynamic plugins directory if needed
+      run: |
+        if [[ "${{ matrix.composeConfig.name }}" == "dynamic-plugins-root" ]]; then
+          mkdir -p dynamic-plugins-root
+        fi
```

```suggestion
    - name: Create dynamic plugins directory if needed
      if: ${{ matrix.composeConfig.name == "dynamic-plugins-root" }}
      run: |
        mkdir -p dynamic-plugins-root
```

nit: Just to avoid running this step completely if not needed and see it skipped in the job view.